stdlib/knative/src/main/ballerina/src/knative/Module.md

```diff
@@ -0,0 +1,30 @@
+## Module Overview
+
+This module offers an annotation based Knative extension implementation for ballerina.
+
+### Annotation Usage Sample:
+
+```ballerina
+import ballerina/http;
+import ballerina/log;
+import ballerina/knative;
+
+@knative:Service {
+    name:"hello"
+}
+listener http:Listener helloEP = new(9090);
+
+@http:ServiceConfig {
+    basePath: "/helloWorld"
+}
+service helloWorld on helloEP {
+    resource function sayHello(http:Caller caller, http:Request request) {
+        http:Response response = new;
+        response.setTextPayload("Hello, World from service helloWorld ! ");
+        var responseResult = caller->respond(response);
```

Comment:
Can be replaced with
```ballerina
var responseResult = caller->respond("Hello, World from service helloWorld ! ");
```

Can we change the variable name to `result` and just use "Hello, world!" as the text?

Reply:
Changes added